test/vc-data-model-1.0/util.js

```diff
@@ -70,10 +70,14 @@ function hasType(doc, expectedType) {
   return type.some(el => el === expectedType);
 }
 
+// https://gist.github.com/marcelotmelo/b67f58a08bee6c2468f8#file-rfc-3339-regex
+const RFC3339regex = /^(\d+)-(0[1-9]|1[012])-(0[1-9]|[12]\d|3[01])\s([01]\d|2[0-3]):([0-5]\d):([0-5]\d|60)(\.\d+)?(([Zz])|([\+|\-]([01]\d|2[0-3])))$/
```

Comment:
I think this regex is missing the `T`/`t` character, see https://gist.github.com/marcelotmelo/b67f58a08bee6c2468f8#gistcomment-2201412

Reply:
Addressed in https://github.com/w3c/vc-test-suite/pull/61/commits/be1829168deaae2f9a9fdf2d6cd4b1f46ff81a8b